frontend/packages/kubevirt-plugin/src/components/vms/vm-console/VMConsoles.tsx

```diff
@@ -139,14 +140,25 @@ const VMConsoles: React.FC<VMConsolesProps> = ({
               {showPassword ? (
                 <>
                   {cloudInitPassword}{' '}
+                  <Button
+                    isSmall
+                    isInline
+                    variant="link"
+                    onClick={() => navigator.clipboard.writeText(cloudInitPassword)}
+                  >
+                    <CopyIcon />
+                    {t('kubevirt-plugin~Copy password')}
```

Comment:
can we move the "copy password" help text to be a pop-over when user hover over the icon, and remove it from regular view ?

Reply:
btw/
can you make it an `Accordion` instead of an `Alert`
https://www.patternfly.org/v4/components/accordion

title of accordion will stay: "Guest login credentials"
but the content will only apear when open
and we can then remove the "Hide/Show password" behaviour because we get it from the accordion

the copy action will be nicer without the "Show/Hide"

Reply:
added, changed the after image in description